### DISCUSSION TITLE
Will it be possible to use LORA 4mb files in webui without merging with models, like hypernetworks?

### DISCUSSION THREAD

**Discussion by sashasubbbb** [d8ahazard/sd_dreambooth_extension#552] [General]
I might be misunderstanding something, but as of right now, you need to merge LORA model with the model it was trained on resulting in a file that’s couple of gigs in size.

**Comment by Thomas-MMJ** [comment]
that will require a patch to web_ui, you can add that as a feature request in the web_ui tracker.

**Comment by sashasubbbb** [comment]
There is some kind of implementation of this feature here https://github.com/kohya-ss/sd-webui-additional-networks 
Unfortunately, it does not support LORA models generated via dreambooth extenstion.

**Reply by Jellybit** [reply]
Thank you for adding this to your comment. I was happy to find that it supports LORA now!
